<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="qwestion_input">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;qwe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Введите вопрос</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qwestion_add">
        <property name="text">
         <string>добавить вопрос</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QLineEdit" name="arg_input">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите аргумент</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="arg_za">
        <property name="text">
         <string>добавить аргумент &quot;за&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arp_prot">
        <property name="text">
         <string>Добавить аргумент &quot;против&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="args_za"/>
      </item>
      <item>
       <widget class="QListWidget" name="args_prot"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLCDNumber" name="lcdNumber_za"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_protiv"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="reset">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 0, 24, 135);</string>
      </property>
      <property name="text">
       <string>сброс</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
